core-services/02-udev.sh

```diff
@@ -2,18 +2,17 @@
 
 [ -n "$VIRTUALIZATION" ] && return 0
 
-if [ -x /usr/lib/systemd/systemd-udevd ]; then
-    _udevd=/usr/lib/systemd/systemd-udevd
-elif [ -x /sbin/udevd -o -x /bin/udevd ]; then
-    _udevd=udevd
-else
-    msg_warn "cannot find udevd!"
-fi
```

Comment:
I assume loosing this systemd-specific file name and path is OK (not that I really personally care)